src/audiovideofacade/DefaultAudioVideoFacade.ts

```diff
@@ -48,8 +48,8 @@ export default class DefaultAudioVideoFacade implements AudioVideoFacade {
     this.audioVideoController.setAudioProfile(audioProfile);
   }
 
-  start(): void {
-    this.audioVideoController.start();
+  start(options?: { signalingOnly?: boolean }): void {
```

Comment:
Benefits of this approach over adding an explicit top-level `prestart()` method?

Reply:
Firstly, it avoids creating an optional method on a core interface -- a customer's `AudioVideoController` subclass doesn't need to change even if the facade does, and that's true even without having to check `if (this.audioVideoController.prestart) { ... }`.

Secondly, this is a one-to-two transition, from "just start" to "maybe there are several kinds or phases of starting", and so I'm looking ahead to the next: check-in mode, full connection but don't start sending media yet, whatever else pops up.

I don't feel strongly about it, and you have both deeper knowledge of the signaling flow and longer experience with the design intent of this API, so happy to receive guidance.

Reply:
Do we see that any app would want to do this as best practice to reduce join time? If so, it doesn't feel like it should be an option.
Is there any plan to change this in future (maybe v3)?

Reply:
Most apps will want to, but knowing _when_ is up to the application.